daemon/logger/fluentd/fluentd.go

```diff
@@ -0,0 +1,40 @@
+package fluentd
+
+import (
+	"github.com/docker/docker/daemon/logger"
+	"github.com/fluent/fluent-logger-golang/fluent"
+)
+
+type Fluentd struct {
+	containerid string
+	writer      *fluent.Fluent
+}
+
+func New(tag string) (logger.Logger, error) {
+	log, err := fluent.New(fluent.Config{FluentPort: 24224, FluentHost: "localhost"})
```

Comment:
There's a PR merged (or about to be merged) that adds `--log-opt` flag to docker; that flag can be used to specify (driver-specific) options, without adding new flags just for this driver.

don't have the PR numbe at hand right now, but @LK4D4 will know